escrow/contracts/escrow.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Unlicensed
+pragma solidity ^0.8.9;
+
+import "@acala-network/contracts/token/Token.sol";
+
+contract Escrow {
+    address payable public requestor;
+    address payable public serviceProvider;
+    address public tokenAddress;
+    uint256 public amount;
+    ServiceStatus public requestorStatus;
+    ServiceStatus public serviceProviderStatus;
+
+    enum ServiceStatus {
+        Pending,
+        Confirmed,
+        Denied
+    }
+
+    constructor(
+        address _tokenAddress,
+        uint256 _amount,
+        address payable _requestor,
+        address payable _serviceProvider
+    ) {
+        requestor = _requestor;
+        serviceProvider = _serviceProvider;
+        tokenAddress = _tokenAddress;
+        amount = _amount;
+        requestorStatus = ServiceStatus.Pending;
+        serviceProviderStatus = ServiceStatus.Pending;
+    }
+
+    function requestorConfirmTaskCompletion(bool _taskConfirmation) public {
```

Comment:
I think this function and `serviceProviderConfirmTaskCompletion` function could be one function with some refactoring.